tests/test_minhash.py

```diff
@@ -1872,6 +1887,30 @@ def test_merge_scaled():
     for k in mh2.hashes:
         assert k in mh3.hashes
 
+def test_add_is_symmetric():
+    mh1 = MinHash(20, 21)
+    mh1.add_hash(5)  
```

Comment:
```suggestion
    mh1.add_hash(5)
```
another unnecessary space ;)